main.py

```diff
@@ -2,14 +2,16 @@
 import discord
 from dotenv import load_dotenv
 
-from cmd import pong, halp, playlist, mal, anichart
+from cmd import pong, halp, playlist, fetchMSG
+from creds import daily, getCreds, clearDatabase
+from creds import purchase, give, setBirthday, checkBirthday
 
 load_dotenv()
 TOKEN = os.getenv('DISCORD_TOKEN')
 GUILD = os.getenv('DISCORD_GUILD')
 
 client = discord.Client()
-MESSAGE_TOKEN = '>'
+MESSAGE_TOKEN = '<'
```

Comment:
should be '>' on master

Reply:
Should i do it before the merge or can i do it after?